gui/src-tauri/Cargo.toml

```diff
@@ -23,19 +23,23 @@ custom-protocol = ["tauri/custom-protocol"]
 [build-dependencies]
 tauri-build = { version = "1.2", features = [] }
 cfg_aliases = "0.1"
+shadow-rs = "0.18"
 
 [dependencies]
 serde_json = "1"
 serde = { version = "1", features = ["derive"] }
-tauri = { version = "1.2", features = ["cli", "devtools", "dialog", "fs-all", "path-all", "shell-execute", "window-close", "window-maximize", "window-minimize", "window-set-resizable", "window-set-title", "window-start-dragging", "window-unmaximize", "window-unminimize"] }
+tauri = { version = "1.2", features = ["devtools", "dialog", "fs-all", "path-all", "shell-execute", "window-close", "window-maximize", "window-minimize", "window-set-resizable", "window-set-title", "window-start-dragging", "window-unmaximize", "window-unminimize"] }
 pretty_env_logger = "0.4"
 log = "0.4"
-clap-verbosity-flag = "1"
+clap = { version = "4.0.29", features = ["derive"] }
+clap-verbosity-flag = "2"
 tauri-plugin-window-state = "0.1.0"
 rand = "0.8.5"
 tempfile = "3"
 which = "4.3.0"
 open = "3"
+shadow-rs = "0.18"
```

Comment:
Need to disable the libgit2 feature. It should just call git from the PATH. Might turn off all default features tbh - tzdb probably is not necessary.